app/views/users/index.html.slim

```diff
@@ -41,4 +41,7 @@ table.users.table.table-striped.table-bordered.table-condensed.table-hover
 
 p.pagination-info
   = page_entries_info @users
-  = paginate @users, theme: 'twitter-bootstrap-3', pagination_class: 'pagination-sm'
\ No newline at end of file
+  = paginate @users, theme: 'twitter-bootstrap-3', pagination_class: 'pagination-sm'
+  / gets this @users from users index
+    .unscope(:order, :select, :where) == User.all
```

Comment:
If it's a WiP, then we know they might or might not go away. Carsten should be able to see the effects of what you're trying to do in a diff tho.